Apps/RaptureIntTests/src/test/java/rapture/lock/LockApiTest.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) 2011-2013 Incapture Technologies LLC
+ *
+ * This is an autogenerated license statement. When copyright notices appear below
+ * this one that copyright supercedes this statement.
+ *
+ * Unless required by applicable law or agreed to in writing, software is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ *
+ * Unless explicit permission obtained in writing this software cannot be distributed.
+ */
+package rapture.lock;
+
+import static org.testng.AssertJUnit.assertNotNull;
+import static org.testng.AssertJUnit.assertNull;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import rapture.common.LockHandle;
+import rapture.common.RaptureLockConfig;
+import rapture.common.RaptureURI;
+import rapture.common.Scheme;
+import rapture.common.client.HttpAdminApi;
+import rapture.common.client.HttpDocApi;
+import rapture.common.client.HttpLockApi;
+import rapture.common.client.HttpLoginApi;
+import rapture.common.impl.jackson.MD5Utils;
+import rapture.helper.IntegrationTestHelper;
+
+
+//TODO: so MUCH repeated code here ... needs a good refactor. JM 04/16/2014
+//TODO: when this (https://incapture.atlassian.net/browse/RAP-2268) is implemented we can clean up the lock auths
```

Comment:
@scarabus could you look at this ticket too https://incapture.atlassian.net/browse/RAP-2268  thanks.